src/libraries/System.IO.Ports/pkg/runtime.native.System.IO.Ports.props

```
@@ -8,7 +8,7 @@
     <DebugType>none</DebugType>
     <IsPackable>true</IsPackable>
     <AllowedOutputExtensionsInSymbolsPackageBuildOutputFolder>$(SymbolsSuffix)</AllowedOutputExtensionsInSymbolsPackageBuildOutputFolder>
-    <TargetsForTfmSpecificDebugSymbolsInPackage>$(TargetsForTfmSpecificDebugSymbolsInPackage);AddRuntimeSpecificNativeSymbolToPackage</TargetsForTfmSpecificDebugSymbolsInPackage>
+    <TargetsForTfmSpecificDebugSymbolsInPackage Condition="'$(keepNativeSymbols)' != 'true'">$(TargetsForTfmSpecificDebugSymbolsInPackage);AddRuntimeSpecificNativeSymbolToPackage</TargetsForTfmSpecificDebugSymbolsInPackage>
```

Is `keepNativeSymbols` an environment variable? Who sets it? Asking as it looks very strange for an msbuild property.

---

It gets set by source build [here](https://github.com/dotnet/runtime/blob/release/6.0/eng/SourceBuild.props#L47)

```
<InnerBuildArgs>$(InnerBuildArgs) /p:KeepNativeSymbols=true</InnerBuildArgs>
```

---

Cool. Should we fix the casing then? :)

```suggestion
    <TargetsForTfmSpecificDebugSymbolsInPackage Condition="'$(KeepNativeSymbols)' != 'true'">$(TargetsForTfmSpecificDebugSymbolsInPackage);AddRuntimeSpecificNativeSymbolToPackage</TargetsForTfmSpecificDebugSymbolsInPackage>
```

---

Maybe also add a comment here explaining what this does?

---

Fixed casing and added comment.